### PR TITLE
[LBGG-533] Account Login & Creation user feedback

### PR DESCRIPTION
**Pull request by buffet-time**
## What

Short description of what was added/changed/removed in this PR

## Link to Issue

Closes https://github.com/leaderboardsgg/leaderboard-site/issues/563

### Steps for testing

- Ensure jest tests pass
- Ensure Login and account creation shows error messages after backend requests

### Images

image
image

### CODE REVIEW

**Comment by zysim** [comment]
Two things:
1. Can we fix the spelling of the login error message? Instead of "Invalid Email, or Password..." let's do "Invalid email or password..."
2. Can we trigger sending from any field on hitting Enter? Right now, I can only log in when I'm in the password field

File: .eslintrc.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
   },
   // add your custom rules here
   "rules": {
+    "no-console": "off",
```
Comment (erunks) [inline .eslintrc.json:48]: Don't know that I particularly agree with turning this off altogether. We definitely don't want to allow `log` or `table` calls to remain in the code, but I see the use for having `warn` or `error` calls. Maybe we do something like this?

```suggestion
    "no-console": ["error", { allow: ["warn", "error"] }],
```

**Review by erunks — APPROVED** [review]
I think this looks good, just the one suggestion. Also I know I did a lot of the API and composable related updates, so it'd probably be good to get another pair of eyes on this. 

Didn't get to AT it though, so I'm only going off code here

**Review by erunks — CHANGES_REQUESTED** [review]
Just noticed this on the netlify preview:
image

The hide/show icon is shifting when the error message is present

**Comment by buffet-time** [comment]
image
moved the `<p>` tag and it renders correctly now

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "lint:fix": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore . --fix",
     "lint:all": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
     "lint": "pnpm lint:all",
-    "precommit": "pnpm lint:all && pnpm test && pnpm pretty:staged ",
+    "lint:prod": "set NODE_ENV=production&& eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
```
Comment (erunks) [inline package.json:19]: Think you missed a space here

```suggestion
    "lint:prod": "set NODE_ENV=production && eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
```

**Review by erunks — COMMENTED** [review]
Just the one comment. Otherwise looks solid.

File: package.json (hunk 2 above)
Comment (buffet-time) [reply package.json:19]: important to note here as well, but as said in the discord. this actually requires a lack of a space here https://stackoverflow.com/a/27090755 comment `use set NODE_ENV=test&& mocha --reporter spec - there is no space between the test and && on purpose.`

I can't actually find why this is how it works but from my testing on windows if we don't include the space it doesn't work. It'll work on Linux but not windows

**Review by erunks — APPROVED** [review]

**Pull request merged**